assignment4/substractionUsingPromise.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Title</title>
+        <script>
+            let number1 = 20;
+            let number2 = 100;
+            let promise = new Promise(function(resolve, reject) {
+                if(number2 <= number1) {
+                    resolve();
+                } else {
+                    reject();
+                }
+            });
+
+            promise.
+            then(function () {
+                console.log(`Substraction = ${number1-number2}`);
+            }).
+            catch(function () {
+                console.log(`Substarction is negative.`);
+            });
```

Comment:
The Assignment is incorrect. Create a function to subtract two numbers. the function should return a promise. based on promise resolve or reject, the result should be shown.